### PR TITLE
Adding toYaml docs

### PR DESCRIPTION
**Pull request by nofearOnline**
@yxxhero 
see:
#1202 
#489

helm main repo issues: 
#4262
#10818

### CODE REVIEW

**Comment by nofearOnline** [comment]
@yxxhero

**Review by yxxhero — APPROVED** [review]
LGTM

**Comment by yxxhero** [comment]
@karenhchu

**Comment by yxxhero** [comment]
@nofearOnline Please fix conflicts.

**Comment by nofearOnline** [comment]
@yxxhero done 😄

**Comment by yxxhero** [comment]
@bacongobbler ping. Thanks very much.

**Comment by yxxhero** [comment]
@bridgetkromhout ping

File: content/en/docs/chart_template_guide/function_list.md [hunk 1]
```diff
@@ -684,9 +684,11 @@ The following type conversion functions are provided by Helm:
   object to indented JSON.
 - `toRawJson` (`mustToRawJson`): Convert list, slice, array, dict, or object to
   JSON with HTML characters unescaped.
+- `toYaml`: Convert list, slice, array, dict, or object to indented yaml, can be used to copy chunks of yaml from any source. This function is equivalent to GoLang yaml.Marshal function, see docs here: https://pkg.go.dev/gopkg.in/yaml.v2#Marshal
 - `fromYaml`: Convert a YAML string to an object.
 - `fromJson`: Convert a JSON string to an object.
 
+
```
Comment (yxxhero) [inline content/en/docs/chart_template_guide/function_list.md:691]: why add this blank line?

**Comment by yxxhero** [comment]
@nofearOnline please fix DCO issue.

File: content/en/docs/chart_template_guide/function_list.md (hunk 1 above)
Comment (nofearOnline) [reply content/en/docs/chart_template_guide/function_list.md]: Got there by accident. I removed it.

**Comment by yxxhero** [comment]
@nofearOnline please fix DCO issue.

**Comment by yxxhero** [comment]
@nofearOnline still has DCO issue.  please see https://github.com/helm/helm-www/pull/1309/checks?check_run_id=9467611315

**Comment by nofearOnline** [comment]
I see the content problems, working on fixing them.

**Comment by yxxhero** [comment]
@nofearOnline Thanks very much.

**Comment by yxxhero** [comment]
@nofearOnline please only add toYaml. I see many changes.

**Comment by nofearOnline** [comment]
@yxxhero done, all should be good now

**Pull request merged**